ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml

```diff
@@ -367,13 +381,38 @@ DFVSC:
   - discharge_fan_power_sensor
   - discharge_fan_speed_frequency_sensor
   - discharge_fan_speed_percentage_sensor
+  - discharge_fan_lost_power_alarm
   uses:
   - discharge_fan_speed_percentage_command
   - discharge_fan_run_status
   - discharge_fan_run_command
   implements:
   - OPERATIONAL
 
+DFVSMC:
+  description: "Variable speed control mode for discharge fans."
+  is_abstract: true
+  opt_uses:
+  - discharge_fan_speed_percentage_command
+  - discharge_fan_speed_frequency_command
+  uses:
+  - discharge_fan_speed_mode
```

Comment:
What does the speed mode do in this context, if you also include discharge_fan_speed points? Does it control its speed continuously or discretely? This point combination feels nonsensical.

Reply:
There is 2 kind of mode whereby the discharege_fan_speed_mode to set low, med, high, vfd (to switch to the percentage command). The percentage_command is used to set the percentage of the speed of the fan.